_inc/lib/tracks/client.php

```diff
@@ -86,15 +86,23 @@ function jetpack_tracks_get_identity( $user_id ) {
 		);
 	}
 
+	if ( 0 === $user_id ) {
```

Comment:
How is this change related to the task in question? It does seem like it could be a bug, of course... maybe we should write a test to cover this?

Reply:
For the same reason we excluded ajax requests (user id 0 for logged out user). We don't want to cookie logged out user.